src/storageManager.js

```diff
@@ -178,6 +178,83 @@ export function newStorageManager({moduleName, moduleType} = {}, {isAllowed = is
     return schedule(cb, STORAGE_TYPE_LOCALSTORAGE, done);
   }
 
+  /**
+   * @returns {boolean}
+   */
+  const sessionStorageIsEnabled = function (done) {
```

Comment:
These are a copy-paste of the local storage methods. I don't have permission to update this PR, but you can find my refactor [here](https://github.com/prebid/Prebid.js/commit/04785e9b0122c4990f108fdb143df246b0732277) (also on the `11928-refactor` branch).

Reply:
Many thanks @dgirardi!
That’s a nice refactoring. I also wondered the same thing but wasn’t sure.
Thanks for doing the branch; we just merged it.